app/controllers/episodes_controller.rb

```diff
@@ -8,4 +8,40 @@ def index
   def show
     @episode = Episode.find(params[:id])
   end
+
+  def edit
+    @episode = Episode.find(params[:id])
```

Comment:
why not to use `before_action` to set `@episode` by `Episode.find(params[:id])` for some methods like `%i[show edit]`?